web/src/components/retro-show/show_retro_page.jsx

```diff
@@ -319,7 +331,7 @@ class ShowRetroPage extends React.Component {
 
     return (
       <span>
-        <RetroWebsocket url={config.websocket_url} retro_id={retroId} websocketRetroDataReceived={this.props.websocketRetroDataReceived}/>
+        <RetroWebsocket url={websocketUrl(config)} retro_id={retroId} websocketRetroDataReceived={this.props.websocketRetroDataReceived}/>
```

Comment:
Can we not just use a relative URL for the websocket? Is this a PWS thing?

Reply:
Depending on the setup it will often run on a different port